lib/challenge.js

```diff
@@ -163,11 +191,14 @@ function prompt8() {
 const starWars = ["Han", "C3PO", "R2D2", "Luke", "Leia", "Anakin"];
 
 function prompt9() {
+  for (let i = 0; i < starWars.length; i++) {
+    console.log('${i}. ${starWars[i]}');
```

Comment:
If you want to use `${}` to console log a variable you have to wrap the whole thing in backticks ` `` ` and not regular quotes